tests/vite-plugin-svelte.ts

```diff
@@ -10,3 +10,7 @@ export async function test(options: RunOptions) {
 		test: ['check:lint', 'check:types', 'test'],
 	})
 }
+
+export const rolldownViteExpectedFailureReason = `
+needs to be updated on vite-plugin-svelte side (tests) and rolldown side (https://github.com/rolldown/rolldown/issues/3403)
```

Comment:
there was one test failing due to the warning log about optimizer esbuild options in vite-plugin-svelte but this has been fixed on main, is there anything else that needs updating there?

Reply:
I meant that warning test. It's still failing on main branch.
https://github.com/vitejs/vite-ecosystem-ci/actions/runs/15601050260/job/43940917304#step:7:843